platform/legacy/__tests__/LegacyPlatformProxifier.test.ts

```diff
@@ -0,0 +1,206 @@
+import { EventEmitter } from 'events';
+import { IncomingMessage, ServerResponse } from 'http';
+
+const mockConfigService = jest.fn(() => ({
```

Comment:
Too much stuff here is just to mock `Root`, maybe we just need `RootMock` or something, but I didn't think about it too much.

Reply:
Yep, it's much better, thanks 👍 `any` to the rescue :)